azure/common/apigee-deployment.yml

```diff
@@ -30,7 +30,7 @@ parameters:
     default: true
   - name: hosted_target_connection_path_suffix
     type: string
-    default: ''
+    default: '/_status'
   - name: hosted_target_healthcheck_suffix
     type: string
```

Comment:
This change should be under **hosted_target_healthcheck_suffix** not  hosted_target_connection_path_suffix